Switch: add getting started tutorial (#4446)
DevExpress/devextreme-documentation: concepts/05 UI Components/Switch/00 Getting Started with Switch/25 Handle Value Change.md

> Users can change a component value (*state*) with a simple click. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#onValueChanged) callback to handle value changes.

Reviewer: "with a simple click" - I assume you can also use touchscreens and the switch supports keyboard too (for accessibility). So either we don't specify how to change the value... Or we should list all the ways to change the value. Otherwise, it looks like the component lacks functionality. Or at least I'm starting to think about (does it NOT support keyboard input?).

BTW, I'm not sure about the SPACE key - so please correct my version if I'm wrong. 

```suggestion
Users can change a component value (*state*) in a number of ways: mouse click, SPACE key, or tap. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#onValueChanged) callback to handle value changes.
```

> Users can change a component value (*state*) in a number of ways: mouse click, SPACE key, or tap. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#onValueChanged) callback to handle value changes.

Reviewer 2: ```suggestion
Users can change a component value (*state*) with a mouse click, SPACE key, or tap. Implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#onValueChanged) callback to handle value changes.
```

The following code displays a notification every time users change the Switch state:

---
##### jQuery

    <!-- tab: index.js -->
    $(function() {
        $("#switch").dxSwitch({
            // ...
            onValueChanged(e) {
                // ...
                const state_label = e.value ? "ON" : "OFF";
                DevExpress.ui.notify(`The component is switched ${state_label}`); 
            }
        });
    });

##### Angular

    <!-- tab: app.component.html -->
    <dx-switch ...
        (onValueChanged)="onValueChanged($event)"
    >
    </dx-switch>

    <!-- tab: app.component.ts -->
    import { Component } from "@angular/core";
    import notify from "devextreme/ui/notify";

    @Component({
        selector: "app-root",
        templateUrl: "./app.component.html",
        styleUrls: ["./app.component.css"]
    })

    export class AppComponent {
        // ...
        onValueChanged(e: {value: boolean}) {
            // ...
            const state_label = e.value ? "ON" : "OFF";
            notify(`The component is switched ${state_label}`); 
        }
    }

##### Vue

    <!-- tab: App.vue -->
    <template>
        <DxSwitch ...

>             @value-changed="onValueChanged($event)"

Reviewer 3: ```suggestion
            @value-changed="onValueChanged"
```
